### ISSUE TITLE
Footer not sticky at bottom

### ISSUE THREAD

**Issue by aitiu** [indiana-university/rivet-shell#2]
![image](https://user-images.githubusercontent.com/19332118/47217806-99d14d80-d377-11e8-93a3-6370d6fe7d9a.png)

**Comment by jolamar** [comment]
For now let's try this until @levimcg has a chance to look at this.

```
body, main {
    height: initial !important;
    min-height: 100vh;
}
```

**Comment by jolamar** [comment]
While you're at it, go ahead and remove the scroll bar from the stage:
```
.rvt-shell__stage {
    overflow-y: initial;
}
```

**Comment by levimcg** [comment]
Just a heads-up on this one. I've got additional off-line reports of this but from a few other folks. It's definitely a priority to get fixed up asap. 🙏

**Comment by zttodd** [comment]
This issue was fixed in PR #19.
